platforms/software/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/transform/ArtifactTransformBuildOperationIntegrationTest.groovy

```diff
@@ -1140,7 +1140,7 @@ class ArtifactTransformBuildOperationIntegrationTest extends AbstractIntegration
         }
     }
 
-    def "project transform execution can be up-to-date or from build cache"() {
+    def "incremental project transform execution can be up-to-date or from build cache"() {
```

Comment:
❌ We still want a test here that a non-incremental project transform execution can be up-to-date or from the build-cache, right?

Reply:
Adding a test for that.